src/_guides/language/effective-dart/style.md

```diff
@@ -323,6 +315,16 @@ defaultTimeout
 kDefaultTimeout
 {% endprettify %}
 
+### DON'T explicitly name libraries
+
+Appending a name to the `library` directive is technically possible,
+but is a legacy feature and discouraged. 
```

Comment:
**issue:** This reads with difficulty. You can say that appending a name to a library has been deprecated.

Reply:
[As I mentioned before](https://github.com/dart-lang/site-www/pull/4390#discussion_r1038649349), nothing about this change is related to a deprecation. You can read the [original issue](https://github.com/dart-lang/language/issues/1073). That's very specific terminology that I don't think should be used interchangeably with "discouraged". 

If you have another suggestion, let me know.